packager/react-packager/src/Resolver/polyfills/Array.prototype.es6.js

```diff
@@ -5,54 +5,51 @@
  * @polyfill
```

Comment:
We don’t read that annotation anywhere. We just add all polyfills as special cases